Dockerfile

```diff
@@ -24,32 +24,44 @@ RUN ln -fs /usr/bin/python3 /usr/local/bin/python \
 RUN go get -u -v sigs.k8s.io/aws-iam-authenticator/cmd/aws-iam-authenticator
 
 # Update all needed tool versions here
-ARG DOCKER_VERSION=18.09.7
-ARG KUBE_VERSION=1.15.6
-ARG HELM_VERSION=2.16.1
-ARG KOPS_VERSION=1.15.0
-ARG POETRY_VERSION=1.0.9
```

Comment:
I distributed these to their installation instructions since editing any of them here invalidates the docker build cache and everything must be re-installed (even if it's just bumping the `POETRY_VERSION`)

Reply:
This would all be made better by using multi-stage builds for these dependencies.

Reply:
:+1: